src/Microsoft.AspNetCore.Razor.Tasks/DotnetToolTask.cs

```diff
@@ -148,19 +148,28 @@ public override void Cancel()
                     }
                     else
                     {
-                        Log.LogMessage(
-                            StandardOutputLoggingImportance,
-                            $"Server execution completed with return code {result}. For more info, check the server log file in the location specified by the RAZORBUILDSERVER_LOG environment variable.");
+                        Log.LogMessage(StandardOutputLoggingImportance, $"Server execution completed with return code {result}. For more info, check the server log file in the location specified by the RAZORBUILDSERVER_LOG environment variable.");
+
+                        if (LogStandardErrorAsError)
+                        {
+                            Log.LogError(completedResponse.ErrorOutput);
+                        }
+                        else
+                        {
+                            Log.LogMessage(StandardErrorLoggingImportance, completedResponse.ErrorOutput);
+                        }
+
+                        return true;
```

Comment:
Doing the same thing as [Roslyn](https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/MSBuildTask/ManagedCompiler.cs#L605) here.